src/components/Molecules/Task/index.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import COLOR from "../../../variables/color";
+import TEXT from "../../../variables/texts";
+import StyledButton from "../../../components/Atoms/EditButton";
+import CheckboxBody from "../../../components/Atoms/Checkbox";
+import StyledInput from "../../../components/Atoms/Input";
```

Comment:
```suggestion
import EditButton from "../../../components/Atoms/EditButton";
import Checkbox from "../../../components/Atoms/Checkbox";
import Input from "../../../components/Atoms/Input";
```
基本コンポーネント名はそのまま呼んでおいた方がいいです！
あとから、「このコンポーネントはどこで使われているんだっけ」と思った時に名前を変えてインポートしていると検索しづらいです。